config/test-ssl.json

```diff
@@ -9,9 +9,9 @@
   "marklogic": {
     "connection": {
       "host": "localhost",
-      "port": 8096,
-      "user": "test-geo-data-services-writer",
-      "password": "test-geo-data-services-writer",
+      "port": 8095,
```

Comment:
This file is only intended though for our tests to make sure that SslTest works. And all of our tests run against port 8096, which is the test app deployed by GDS. 

We can also have SSL sample config in default.json, which would point to 8095... though even that's a little pointless since the odds of a user having an ML app already on port 8095 are extremely remote.

Reply:
Ok. We can leave it the way it is.
I was using this config to try things out with ArcGIS and QGIS. Initially I was confused because the data wasn't as expected since I originally used the non-ssl connection which points to ML 8095. So, I thought maybe we should make them uniform.